WORKSPACE

```diff
@@ -14,15 +14,14 @@
 
 workspace(name = "build_bazel_rules_typescript")
 
-# Using a pre-release snapshot to pick up a commit that makes all nodejs_binary
-# programs produce source-mapped stack traces.
-RULES_NODEJS_VERSION = "926349cea4cd360afcd5647ccdd09d2d2fb471aa"
+# Using a pre-release snapshot to pick up a fix for puppeteer
+RULES_NODEJS_VERSION = "0162fdbe8ed986c9b5d5b79e53c98385ddaf6edd"
```

Comment:
don't think you need this anymore?

Reply:
you mean switch to git_repository from the skylark rules?

Reply:
nevermind, this is just a fast-forward of master